listener/avalon_listener/tcs_listener.py

```diff
@@ -257,38 +287,56 @@ def parse_command_line(config, args):
         config['Logging']['LogFile'] = options.logfile
     if options.loglevel:
         config['Logging']['LogLevel'] = options.loglevel.upper()
-    if options.bind_uri:
-        bind_uri = options.bind_uri
+    if options.bind:
+        host_name, port = parse_bind_url(options.bind)
+    else:
+        if config.get("Listener") is None or \
+                config["Listener"].get("bind") is None:
+                    logger.warn("quit due to no suitable config for Listener")
+                    sys.exit(-1)
+        host_name, port = parse_bind_url(
+            config["Listener"].get("bind"))
     if options.lmdb_url:
-        config["KvStorage"]["remote_url"] = options.lmdb_url
+        config["KvStorage"]["remote_storage_url"] = options.lmdb_url
```

Comment:
If lmdb url is not present log error and exit

Reply:
ok